nf_core/pipeline-template/.github/workflows/ci.yml

```
@@ -11,14 +11,16 @@ on:
 
 env:
   NXF_ANSI_LOG: false
+  NXF_SINGULARITY_CACHEDIR: {% raw %}${{ github.workspace }}{% endraw %}/.singularity
+  NXF_SINGULARITY_LIBRARYDIR: {% raw %}${{ github.workspace }}{% endraw %}/.singularity
 
-concurrency:
-  group: "{% raw %}${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}{% endraw %}"
+  concurrency:
```

```suggestion
concurrency:
```
to fix the linting error

---

:facepalm:

---

that's not it